src/components/app/albumsongscontainer/songplayer/song-player.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+
+@inject('songsStore')
+@observer
+class SongPlayer extends Component {
+  constructor(props) {
+    super(props)
+
+    this.props.songsStore.getClickedSong()
+  }
+
+  componentWillReceiveProps() {
+    this.props.songsStore.getClickedSong()
+    this.refs.player.pause()
+    this.refs.player.load()
+  }
+
+  render() {
+    const { songsStore } = this.props
+
+    return (
+      <div className="sp__container">
+        <span className="sp__song-name">{songsStore.songToDisplay.name}</span>
+        <audio controls="controls" ref="player" className="sp__player">
```

Comment:
String refs są deprecated :P Użyj React.createRef

Reply:
eh no wlasnie wiem ze sa. Totalnie mi sie to nie podoba, bo bardziej mi sie podobaly. XD done

Reply:
Nie no lipne były :P Zawsze możesz się dupnąć przy pisaniu i się apka wywali + wszystko było globalne, a tak to masz zmienna i zawsze wiesz skąd co się wzięło :P